app/directives/group-page-stats/group-page-stats.coffee

```diff
@@ -0,0 +1,14 @@
+null
+
+### @ngInject ###
+global.cobudgetApp.directive 'groupPageStats', () ->
+    restrict: 'E'
+    template: require('./group-page-stats.html')
+    replace: true
+    controller: ($scope, $location) ->
+
+     $scope.showBucket = (bucketId) ->
+       $location.path("/buckets/#{bucketId}")
+
```

Comment:
this is already defined in `group-page-buckets.coffee`. presumably there is a place we could define this such that it would be available to both directives?